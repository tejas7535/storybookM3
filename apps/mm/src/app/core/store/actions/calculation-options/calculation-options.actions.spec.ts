import { PreflightData } from '@mm/core/services/preflght-data-parser/preflight-data.interface';

import { CalculationOptionsActions } from '..';

describe('CalculationOptionsActions', () => {
  it('should create an action to fetch preflight options', () => {
    expect(CalculationOptionsActions.fetchPreflightOptions()).toMatchSnapshot();
  });

  it('should create an action to set preflight options', () => {
    expect(
      CalculationOptionsActions.setCalculationOptions({
        options: {
          innerRingExpansion: '0.35',
        } as Partial<PreflightData> as PreflightData,
      })
    ).toMatchSnapshot();
  });
});
