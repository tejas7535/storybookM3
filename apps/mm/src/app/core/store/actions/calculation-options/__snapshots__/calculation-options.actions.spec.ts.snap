// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalculationOptionsActions should create an action to fetch preflight options 1`] = `
Object {
  "type": "[CalculationOptions] Fetch preflight options",
}
`;

exports[`CalculationOptionsActions should create an action to set preflight options 1`] = `
Object {
  "options": Object {
    "innerRingExpansion": "0.35",
  },
  "type": "[CalculationOptions] Set Options",
}
`;
